test/functional/services/remote/webdriver.ts

```diff
@@ -203,12 +201,7 @@ async function attemptToCreateCommand(
             .build();
           return {
             session,
-            consoleLog$: pollForLogEntry$(
-              session,
-              logging.Type.BROWSER,
-              config.logPollingMs,
-              lifecycle.cleanup.after$
-            ).pipe(
+            consoleLog$: pollForLogEntry$(session, logging.Type.BROWSER, config.logPollingMs).pipe(
               takeUntil(lifecycle.cleanup.after$),
```

Comment:
Wouldn't that row need to be updated to `takeWhile` as well?

Reply:
Right, I missed that. Thank you